### PR TITLE
Add sample configuration to use AWS Marketplace

### PR DESCRIPTION
**Pull request by kota2and3kan**
## Description

This PR adds the sample configuration when you use AWS Marketplace. The configuration is sample, so I added it as a commented out configuration.

Please take a look!

## Related issues and/or PRs

* https://github.com/scalar-labs/docs-internal-scalardl/pull/62
  * I updated the document related to these configurations.

## Changes made

* Add commented out configurations of AWS Marketplace.

## Checklist

- [x] I have updated the Japanese version of this documentation.
- [x] I have updated all applicable versions of the documentation.
- [x] I have checked that my documentation looks as expected on a locally built version of the docs site. (See [Build and serve the staged docs site locally](https://github.com/scalar-labs/docs-internal-scalardl/blob/main/README.md#build-and-serve-the-staged-docs-site-locally))
- [x] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
- [x] Any dependent changes in other PRs have been merged and published.

## Additional notes (optional)

N/A

### CODE REVIEW

File: docker-compose-auditor.yml [hunk 1]
```diff
@@ -73,6 +73,7 @@ services:
 
   scalar-auditor:
     image: ghcr.io/scalar-labs/scalar-auditor:4.0.0-SNAPSHOT
+    # image: 709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalar-auditor:4.0.0-SNAPSHOT
```
Comment (jnmt) [inline docker-compose-auditor.yml:76]: Sorry, just my curiosity. Basically, is this part `709825985650 ` never change? Is there any stable name just like ghcr.io ...?

**Review by jnmt — APPROVED** [review]
LGTM! Thank you. I left a minor question, but it doesn't block merging. BTW, we might need to think about the default branch or documentation since users might use non-released versions unintentionally.

File: docker-compose-auditor.yml (hunk 1 above)
Comment (kota2and3kan) [reply docker-compose-auditor.yml:76]: @jnmt  
Thank you for your review!

> Sorry, just my curiosity. Basically, is this part `709825985650` never change? Is there any stable name just like ghcr.io ...?

Yes. The part `709825985650` is fixed value. It depends on the specifications of AWS Marketplace.
If you use Scalar products via AWS Marketplace, you can always use the same value `709825985650`.

**Comment by kota2and3kan** [comment]
@jnmt  
> BTW, we might need to think about the default branch or documentation since users might use non-released versions unintentionally.

Ah, I see. Your concern is right. I will update the document to add a step that change the branch to specific version on the document side.

**Comment by kota2and3kan** [comment]
@jnmt 

> > BTW, we might need to think about the default branch or documentation since users might use non-released versions unintentionally.
> 
> Ah, I see. Your concern is right. I will update the document to add a step that change the branch to specific version on the document side.

I updated the document in https://github.com/scalar-labs/docs-internal-scalardl/pull/62/commits/e9993384a600931edf0b41ab89e9de9ee2efce5e on the document PR side.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**